packages/components/socialFeed/RichText/utils.ts

```diff
@@ -0,0 +1,55 @@
+export const fixLinkOnHTMLString = (htmlString: string) => {
```

Comment:
This fix has been only implemented for apps only. 
There are few issues on HTML of the article; this should be fixed on the Article creation

```<ul>
  <li>Website: https://foxinodes.net</li>
  <li>
    Discord Server:
    <a href="https://discord.gg/uATTFEFRGx">https://discord.gg/uATTFEFRGx</a>
  </li>
  <li>X (Twitter): https://twitter.com/foxinodes</li>
  <li>
    Medium:
    <a href="https://medium.com/@Foxinodes">https://medium.com/@Foxinodes</a>
  </li>
</ul>
```
Here links are not placed inside <a> tag
Reference post: https://app.teritori.com/feed/post/tori-9edd0333-e427-4b8b-8fa8-49ef72c73b94

Reply:
thank you for this. I should create a different ticket for the web issue.